src/views/index.html

```diff
@@ -1,107 +1,97 @@
-          A news site scraping API with a few different endpoints that provide
-          links to articles related to energy prices.
-        </p>
-        <p>
-          This project's secondary aim is to help beginners learn how to
-          contribute to open source projects and will be purposely kept as basic
-          as possible.
-        </p>
-      </article>
-      <article>
-        <h2>Planned Features</h2>
         <ul>
-          <li>Links to articles</li>
-          <li>End points for specific sources</li>
-          <li>Thorough documentation</li>
-          <li>
-            An open and inviting issue-board to encourage discussion of future
-            features
-          </li>
+            <li><a href="/">Home</a></li>
+            <li><a href="/api/news">API</a></li>
+            <li>
+                <a href="https://github.com/MizouziE/energy-prices-api">Docs</a>
+            </li>
+            <li>
+                <a href="https://github.com/MizouziE/energy-prices-api">About</a>
+            </li>
         </ul>
-      </article>
-      <article>
-        <h2>Try it now!</h2>
-        <p>
-          Access to the
-          <a
-            href="https://github.com/MizouziE/energy-prices-api/blob/master/README.md"
-            >documentation.</a
+    </nav>
+    <main>
+        <article>
+            <h2>What is this?</h2>
+            <p>
+                A news site scraping API with a few different endpoints that provide links to articles related to energy prices.
+            </p>
+            <p>
+                This project's secondary aim is to help beginners learn how to contribute to open source projects and will be purposely kept as basic as possible.
+            </p>
+        </article>
+        <article>
+            <h2>Planned Features</h2>
+            <ul>
+                <li>Links to articles</li>
+                <li>End points for specific sources</li>
+                <li>Thorough documentation</li>
+                <li>
+                    An open and inviting issue-board to encourage discussion of future features
+                </li>
+            </ul>
+        </article>
+        <article>
+            <h2>Try it now!</h2>
+            <p>
+                Access to the
+                <a href="https://github.com/MizouziE/energy-prices-api/blob/master/README.md">documentation.</a
           >
         </p>
         <ul>
           <li>
             Endpoint to the News data results -> <a href="/api/news">here!</a>
-          </li>
-          <li>
+                </li>
+                <li>
+                    Endpoint to the News Sources data results -> <a href="/api/news/sources">here!</a>
+                </li>
+                <li>
+                    Endpoint to the News Regions results -> <a href="/api/news/regions">here!</a>
+                </li>
+                <li>
+                    Endpoint to the News Regions(id) results -> <a href="/api/news/regions/:id">here!</a>
```

Comment:
This would need an example endpoint with a brief explanation that it **is only** an example. I think this link would throw a 404.